packages/media/src/getCanvasBlob.ts

```diff
@@ -14,11 +14,11 @@
  * limitations under the License.
  */
 
-function getCanvasBlob(canvasEl) {
+function getCanvasBlob(canvasEl: HTMLCanvasElement): Promise<Blob | null> {
   return new Promise(function (resolve) {
     canvasEl.toBlob((blob) => {
-      resolve(blob, 'image/jpeg');
-    });
+      resolve(blob);
+    }, 'image/jpeg');
```

Comment:
<!-- highlight something positive -->
**👏 praise:**

Nice catch!

Reply:
Looks like this actually fixes #12132 too!